CountingSort/app_resources/prefix_sum_shader.comp.hlsl

```diff
@@ -0,0 +1,53 @@
+#include "nbl/builtin/hlsl/bda/__ptr.hlsl"
+#include "nbl/builtin/hlsl/sort/counting.hlsl"
+#include "nbl/builtin/hlsl/sort/common.hlsl"
+
+[[vk::push_constant]] nbl::hlsl::sort::CountingPushData pushData;
+
+struct PtrAccessor
+{
+    static PtrAccessor createAccessor(uint64_t addr)
```

Comment:
we have a convention of calling HLSL-pretend-constructors `create` always

Reply:
Done

Reply:
Done